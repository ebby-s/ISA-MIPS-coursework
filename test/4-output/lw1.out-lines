TB : INFO : register_v0=2694000a
TB : finished; active=0
