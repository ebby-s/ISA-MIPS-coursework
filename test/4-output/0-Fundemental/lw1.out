TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x8c620004
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00000000
TB : INFO : register_v0=00000050
TB : Finished : active=0
